pkg/webui/console/store/middleware/logics/pubsubs.js

```diff
@@ -19,13 +19,23 @@ import createRequestLogic from '@ttn-lw/lib/store/logics/create-request-logic'
 import * as pubsubs from '@console/store/actions/pubsubs'
 import * as pubsubFormats from '@console/store/actions/pubsub-formats'
 
+const createPubsubLogic = createRequestLogic({
+  type: pubsubs.CREATE_PUBSUB,
+  process: async ({ action }) => {
+    const { appId, pubsub } = action.payload
+
+    return tts.Applications.PubSubs.create(appId, pubsub)
```

Comment:
```suggestion
    return await tts.Applications.PubSubs.create(appId, pubsub)
```
You have to use await here. It might seem redundant but actually has a different effect when used within a try/catch block. If you return an erroring promise directly, it will return the pending promise. In this case, it would mean that the error handling within `createRequestLogic()` would be circumvented. Using `await` will make sure that the promise evaluates before being returned and any error along the way will be thrown before returning the function. 
See this post on SO for more info https://stackoverflow.com/a/42750371/3944120